test/packages/test_networkx.py

```diff
@@ -0,0 +1,26 @@
+from textwrap import dedent
+
+import pytest
+
+
+def test_networkx_basicgraph(selenium_standalone, request):
+    selenium = selenium_standalone
+
+    if selenium.browser == 'chrome':
+        request.applymarker(pytest.mark.xfail(
+            run=False, reason='chrome not supported'))
```

Comment:
Is this necessary? Looks like this test passed on chrome,
```
XPASS test/packages/test_networkx.py::test_networkx_basicgraph[chrome] chrome not supported
```
other tests that need it are mostly large packages with lots of C extensions (pandas, scipy)..

Reply:
I see, thanks for the clarification, I thought it was just required boilerplate for some reason